develop/app/Controllers/v2/WalletController.php

```diff
@@ -54,7 +53,7 @@ public function show()
     public function create()
     {
         $data = $this->request->getJSON(true);
-
+        
```

Comment:
What does this new line do?

Reply:
In the original show method, there was a comment to mark the arguments to be passed in the show method, but after the show method was modified, there was no need to pass the arguments, so the comment was removed from this line,So the code goes up one line.